port/unix/omrsysinfo.c

```diff
@@ -3617,6 +3655,63 @@ readCgroupSubsystemFile(struct OMRPortLibrary *portLibrary, uint64_t subsystemFl
 	return rc;
 }
 
+/** 
+ * Checks if the process is running inside docker container 
+ *
+ * @param[in] portLibrary pointer to OMRPortLibrary
+ * @param[out] inDocker pointer to BOOLEAN which on successful return indicates if the process is running in docker container or not 
+ *
+ * @return 0 on success, otherwise negative error code
+ */
+static int32_t
+isRunningInDocker(struct OMRPortLibrary *portLibrary, BOOLEAN *inDocker)
```

Comment:
Should this be `isRunningInContainer`?  It appears to generically check for a container runtime, not docker specifically.

Reply:
`isRunningInContainer` is also fine. The mechanism works for runc based containers, not specific to docker.